secure/storage/src/tests/vault.rs

```diff
@@ -9,11 +9,28 @@ use crate::{
 use libra_crypto::{test_utils::TestLibraCrypto, Signature};
 use libra_vault_client::dev::{self, ROOT_TOKEN};
 
-/// VaultStorage test constants
+/// VaultStorage namespace constants
 const VAULT_NAMESPACE_1: &str = "namespace_1";
 const VAULT_NAMESPACE_2: &str = "namespace_2";
 const VAULT_NAMESPACE_3: &str = "namespace_3";
 
+/// VaultStorage KV data names
+const ANYONE: &str = "anyone";
```

Comment:
why? these are only used in one test ... global constants should only be used when you want reuse. I don't want reuse :P

Reply:
I can move the constants to be scoped only for the test, but either way, a constant is better than having repeated strings throughout the code 😄 Although, I also re-used PARTIAL for the token tests so it is kind of global.. I'll leave as is for now so I can land but if you have a better suggestion, I'll happily follow up in a PR!